R/teal_slice.R

```diff
@@ -483,44 +520,51 @@ c.teal_slices <- function(...) {
   )
 }
 
-
-# format method for teal_slices
+# store method for `teal_slices` object
+#' @param x `teal_slices` object
+#' @param file a `character(1)` with the storage destination name
 #' @export
 #' @rdname teal_slice
 #' @keywords internal
 #'
-format.teal_slices <- function(x, show_all = FALSE, ...) {
-  res <- character(0)
-  for (i in seq_along(x)) {
-    ind <- names(x)[i]
-    if (is.null(ind)) ind <- sprintf("[[%d]]", i)
-    res <- append(res, ind)
-    res <- append(res, format(x[[i]], show_all = show_all, ...))
-  }
+store_filters <- function(x, file, ...) {
```

Comment:
```suggestion
store_filters <- function(tss, file, ...) {
```
I prefer `store_slices` myself, it keeps in line with the other functions in this family.